src/main/java/de/uni_tuebingen/ub/nppm/model/EinzelbelegHatStand.java

```diff
@@ -0,0 +1,45 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "einzelbeleg_hatstand")
+public class EinzelbelegHatStand {
```

Comment:
All other relation-related tables are named with a suffix called _MM. Should this also be added here? (e.g. similar to EinzelbelegHatEthnie_MM, EinzelbelegFatHunktion_MM, and others)

Reply:
No